docker-for-aws/faqs.md

```diff
@@ -137,6 +137,20 @@ For more information about adding an SSH key pair to your account, please refer
 
 All container logs are aggregated within [AWS CloudWatch](https://aws.amazon.com/cloudwatch/).
 
+## Best practice to deploy a large cluster
+
+When deploying a cluster of more than 20 workers, it can take a very long time for AWS to deploy all of the instances (1+hrs).
+It is best to deploy a cluster of 20 workers then scale it up in the Auto-Scaling Group (ASG) once it's been deployed.
```

Comment:
We shouldn't recommend that they change the ASG directly, this could cause problems on upgrades. They should update the cloudformation stack, and change the number of workers. This is safer.

Reply:
The problem is the time difference between the deployments. What's the point of having the ASG if they can't change it there directly. 
Especially since we recommend scaling that way:
https://docs.docker.com/docker-for-aws/scaling/#scaling-workers

Reply:
If you do an update on the cloud formation stack, it is still going to change the ASG behind the scenes but it keeps the stack in sync with the change, and this will prevent issues in the future, if they decide to upgrade. 

We should update the docs to remove the second option, to remove confusion.

Reply:
@kencochrane what's the exact failure scenario? This should work fine, no?

 1. Start swarm with 20 workers
 2. Scale up to 200 workers
 3. Apply an stack update (making sure that the worker count is still correct)

Reply:
@friism I believe that would work, but it would depend on the person who is updating the stack to make sure the worker count value is set the same as the ASG value. If they don't do that, they could be surprised.

Reply:
yes, I agree that's a bit precarious, and we should call that out in the docs. But it seems like a more reliable approach than trying to vault a 200-node swarm into existence, encountering timeouts and other problems.

wdyt?

Reply:
OK, I just tried it out, this is what I found.

If I create a stack with 0 workers. I then changed the ASG manually to 3, waited for the 3 workers to join the swarm. Then I ran a stack update, and set the worker to 3. The stack was updated, and the 3 worker nodes didn't change. This confirmed what you were saying @friism 

I then tested the following. I started a stack, set the workers to 0. I manually changed the ASG to 3 workers, and waited for them to join the swarm. I then updated the stack and set the worker count to 2. This resulted in the ASG to remove one of the nodes so that I was down to 2 workers to match what the stack update had. Which is what I was expecting.

I'm OK with leaving it the way it is, and then we just update the docs, but I'm afraid that people don't read the docs and they might get burned. Not much we can do about that, but just wanted to point it out.

Reply:
Thanks for testing @kencochrane. I agree we should point out that upgrade pitfall

Reply:
Essentially you need to set your stack update to the number currently running, correct?
I can add that to the docs

Reply:
Yup